Python 3.8.10 (default, Mar 15 2022, 12:22:08) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> say_my_name = __import__('3-say_my_name').say_my_name
>>> say_my_name("Abdulquyum", "Ajumobi")
My name is Abdulquyum Ajumobi
>>> say_my_name(20, "Ajumobi")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/3-say_my_name.py", line 4, in say_my_name
    raise TypeError("first_name must be a string")
TypeError: first_name must be a string
>>> say_my_name("Abdulquyum", 54)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/3-say_my_name.py", line 6, in say_my_name
    raise TypeError("last_name must be a string")
TypeError: last_name must be a string
>>> say_my_name("Abdulquyum", True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/3-say_my_name.py", line 6, in say_my_name
    raise TypeError("last_name must be a string")
TypeError: last_name must be a string
